pom.xml

```diff
@@ -86,18 +86,9 @@
                             </excludes>
                         </filter>
                     </filters>
-                    <shadedArtifactAttached>false</shadedArtifactAttached>
-                    <createDependencyReducedPom>true</createDependencyReducedPom>
-                    <relocations>
-                        <relocation>
-                            <pattern>com.fasterxml.jackson</pattern>
-                            <shadedPattern>rockset.com.fasterxml.jackson</shadedPattern>
-                        </relocation>
-                        <relocation>
-                            <pattern>com.google.gson</pattern>
-                            <shadedPattern>rockset.com.google.gson</shadedPattern>
-                        </relocation>
-                    </relocations>
+                    <shadedArtifactAttached>true</shadedArtifactAttached>
+                    <shadedClassifierName>uberjar</shadedClassifierName>
```

Comment:
Should we call this jdbc or something? Ok, either way.

Reply:
I don't think so. JDBC doesn't necessities uberjars - if a user wants to use jdbc to talk to Rockset in a project using maven, they don't need it. It's the BI tools that want uberjars, which happen to use the JDBC interface.

Reply:
yeah this has both client and jdbc which we should separate in the future